iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/VoltageLimitsAdderImpl.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.impl;
+
+import com.powsybl.iidm.network.*;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+class VoltageLimitsAdderImpl implements VoltageLimitsAdder {
+
+    private final OperationalLimitsOwner owner;
+    private double highVoltage = Double.NaN;
+    private double lowVoltage = Double.NaN;
+
+    VoltageLimitsAdderImpl(OperationalLimitsOwner owner) {
+        this.owner = owner;
+    }
+
+    @Override
+    public VoltageLimitsAdder setHighVoltage(double highVoltage) {
+        this.highVoltage = highVoltage;
+        return this;
+    }
+
+    @Override
+    public VoltageLimitsAdder setLowVoltage(double lowVoltage) {
+        this.lowVoltage = lowVoltage;
+        return this;
+    }
+
+    @Override
+    public VoltageLimits add() {
```

Comment:
To be discuss: is it better to return the VoltageLimits or to return the owner to be able to create additional limits?

Reply:
Both approaches are good for me. In favor of returning the limits: this is what we are doing with reactive limits so it would be consistent regarding the API. Furthermore, not sure it makes sense for VoltageLimits for example: objects supporting voltage limits do not support other kinds of limits.